.gitignore

```diff
@@ -3,6 +3,9 @@
 # dependencies
 /node_modules
 
+#mapbox key
+src/YourKey.js
+
```

Comment:
Is this required?
If it is not required, please remove these lines.

Reply:
It was used for mapbox-user-token which has not to be opened. I removed this code as the method of delivering user token has been changed.